src/popover.jsx

```diff
@@ -104,21 +104,39 @@ export default class Popover extends React.Component {
   }
 
   render() {
-    let toggleButton = React.cloneElement(this.props.toggleButton, {
-      ref: 'toggleButton'
-    });
+    if(this.props.toggleButton){
+      var toggleButton = React.cloneElement(this.props.toggleButton, {
+        ref: 'toggleButton',
```

Comment:
Do we need a ref to this? If not let's remove it. React complains a lot about these things these days.